src/slider/slider.js

```diff
@@ -83,16 +83,6 @@
   };
 
   /**
-   * Handle mouseup on element.
-   *
-   * @param {Event} event The event that fired.
-   * @private
-   */
-  MaterialSlider.prototype.onMouseUp_ = function(event) {
-    event.target.blur();
```

Comment:
I'll test it and see if that actually works.

tbh I feel personally that blurring is wrong since you lose the focus state which is important for a11y to know where you are in the page for tab ordering. But, we've done it since launch so best to try and break things as little as possible.